sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -1328,6 +1332,155 @@ void captureSessionToken(RxDocumentServiceRequest request, RxDocumentServiceResp
         return queryDocuments(collectionLink, "SELECT * FROM r", options);
     }
 
+    @Override
+    public Mono<FeedResponse<Document>> readMany(
+        List<Pair<String, PartitionKey>> itemKeyList,
+        String collectionLink) {
+        
+        DocumentCollection collection = 
+            this.readCollection(collectionLink, new RequestOptions()).block().getResource();
```

Comment:
is this some leftover code? is this needed?

Reply:
Yup. Deleted